networking-calico/networking_calico/plugins/ml2/drivers/calico/test/test_plugin_etcd.py

```diff
@@ -121,6 +121,9 @@ def etcd3gw_client_put(self, key, value, **kwargs):
                 if 'labels' in self.recent_writes[key]['metadata']:
                     existing_v3_metadata['labels'] = \
                         self.recent_writes[key]['metadata']['labels']
+                if 'annotations' in self.recent_writes[key]['metadata']:
+                    existing_v3_metadata['annotations'] = \
+                        self.recent_writes[key]['metadata']['annotations']
```

Comment:
Is this a leftover from an earlier idea?

Reply:
Sorry, my mistake.

However, optionally - if you don't mind another cycle of review - it would be better to update the comment above to say:
```
            # If this is an update, check that the metadata, other than labels
            # and annotations, is unchanged.
```

Reply:
Sure, updated comment.